modules/tinymce/src/core/main/ts/api/Options.ts

```diff
@@ -1009,13 +1035,17 @@ const shouldConvertUnsafeEmbeds = option('convert_unsafe_embeds');
 const getLicenseKey = option('license_key');
 const getApiKey = option('api_key');
 const isDisabled = option('disabled');
+const allowExtraMathmlAttributes = option('allow_extra_mathml_attributes');
+const allowExtraMathmlElements = option('allow_extra_mathml_elements');
```

Comment:
Could we prefix these with `get` so they are `getAllowExtraMathmlAttributes` since the other getters are prefixed like that or `is` if they return a boolean.

Reply:
Sure.